src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -1701,6 +1701,15 @@ public GHPullRequestQueryBuilder queryPullRequests() {
         return new GHPullRequestQueryBuilder(this);
     }
 
+    /**
+     * Retrieves pull requests according to search terms.
+     *
+     * @return gh pull request search builder for current repository
+     */
+    public GHPullRequestSearchBuilder searchPullRequests() {
+        return new GHPullRequestSearchBuilder(this);
```

Comment:
```suggestion
        return new GHPullRequestSearchBuilder(this.root).repo(this);
```

Reply:
Was about to fix today, but you already did it 👍